fearless/Modules/NewWallet/WalletSend/WalletSendPresenter.swift

```diff
@@ -56,7 +59,8 @@ final class WalletSendPresenter {
             tipRequired: chainAsset.chain.isTipRequired,
             tipViewModel: provideTipViewModel(),
             feeViewModel: provideFeeViewModel(),
-            amountInputViewModel: provideInputViewModel()
+            amountInputViewModel: provideInputViewModel(),
+            scamInfo: scamInfo
```

Comment:
You put it to viewModel just to get it from this viewModel without any changes in the "bind" method. You can just take it from own properties directly.

Reply:
this is your module, why you don't refactor WalletSendViewState for receive separately viewModels?